cmd/webui/main.go

```diff
@@ -115,9 +126,10 @@ func addPingRoute(r *mux.Router) {
 	})
 }
 
-func addAssetsPath(r *mux.Router, assetPath string) {
+func addAssetsPath(r *mux.Router, assetPath string) error {
 	if _, err := os.Stat(assetPath); err != nil {
-		panic(fmt.Sprintf("ERROR: unable to stat %s: %v", assetPath, err))
+		return errors.Wrap(err, fmt.Sprintf("ERROR: unable to stat %s", assetPath))
```

Comment:
Could we do `fmt.Errorf` instead with `%w`? It'll drop the need for a new dependency.